ironic/drivers/modules/drac/raid.py

```diff
@@ -294,6 +294,44 @@ def clear_foreign_config(node, raid_controller):
         raise exception.DracOperationError(error=exc)
 
 
+def change_physical_disk_state(node, mode,
+                               controllers_to_physical_disk_ids=None):
+    """
+    :param node: an ironic node object.
+    :param mode: enumeration that indicates the mode
+                     to change the disks to.
+    :param controllers_to_physical_disk_ids:Dictionary of controllers and
+                                            corresponding disk ids to convert
+                                            to the requested mode
+    :return: a dictionary containing:
+             - conversion_results, a dictionary that maps controller ids
+             to the conversion results for that controller.  The
```

Comment:
Can you make the formatting for conversion results be the same as it is in the _change_physical_disk_conversion() method below?

Reply:
Done